index.js

```diff
@@ -45,6 +47,11 @@ console.log('hello world')
  *
  * Example Output: "$49.90"
  */
+let price = 9.98;
+let quantity = 5;
+let total = price * quantity;
```

Comment:
What if the prices you were given was 9.985? and you needed to round it to two places? Probably not very likely with groceries, but micro-investments, and lots of other data is definitely handled this way, anyway, i'd change your number and round it to two decimals before i printed it out.